### DISCUSSION TITLE
NCUEDU-AI是如何判斷文章或是回覆具有獨創性的呢？

### DISCUSSION THREAD

**Discussion by alvin1226** [1101-GS3073-C/Python-Community-GS3073-C#687] [General]
我覺得應該跟字數沒關係，我看過字很少有被給讚，也有字很多沒被給的
我猜應該是把這邊的文章和回覆拿回去判斷，重複性較低的會給？
我把這篇作為實驗，我想應該沒有人有問過類似的吧

**Comment by alvin1226** [comment]
欸果然這篇就被給了，看來好像可以看一下其他人發的來增加自己被給讚的機會摟！

**Comment by liangyinglly** [comment]
我也很好奇他是怎麼給讚跟眼睛的

**Comment by Vincent20030314** [comment]
這真是個令人匪夷所思的問題。

**Comment by HsuChen88** [comment]
感覺只要是它沒看過的就越容易拿讚，反之籠統的說法會很常見，就很容易拿眼睛。

**Comment by Valentinaa0214** [comment]
我也猜是拿去跟之前的文章做重複性的比較來判斷的，
但是兩班(Python班跟AI班)的討論區內容似乎是不互通的，文章雖然重複但還是都會獲得讚！
可能留言的讚跟眼睛是跟同一篇的其他留言相互比較嗎，然後看有沒有新的觀點

**Comment by jerry0108566** [comment]
那要多讚它才會給讚呢？

**Comment by jerry0108566** [comment]
我剛po了一篇文，打了一長串，結果收到了眼睛。
後來我全刪掉，只留一句個人觀點反而眼睛消失了。
真的挺玄的。
